packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -473,6 +472,28 @@ export class Users {
       }
     );
 
+    const derivedDbMarketsInfos = await db.findMarkets({
+      selector: { market: { $in: marketIds } },
+    });
+    for (const derivedMarketDbInfo of derivedDbMarketsInfos) {
+      if (derivedMarketDbInfo.reportingState === MarketReportingState.Finalized || MarketReportingState.AwaitingFinalization) {
+        const derivedDbDisputeInfo = await db.findDisputeDocs({
+          selector: { market: { $eq: derivedMarketDbInfo.market } },
+        });
+        const reportingFeeDivisor = derivedMarketDbInfo.feeDivisor;
+        // TODO: set numberOfShares based on winning/tentative payout
+        const reportingFee = numberOfShares.div(reportingFeeDivisor);
+        const contracts = augur.contracts;
+        const totalUnclaimedProceeds = (await contracts.claimTradingProceeds.calculateProceeds_(derivedMarketDbInfo.market, new BigNumber(tentativeOutcome), numberOfShares))
```

Comment:
This is going to do a call per market. We should add this information to the logs instead

Reply:
Alternatively you can calculate this with the existing data in the market using the winningPayoutNumerators pretty easily.

Reply:
True, but if a market is awaiting finalization, it won't be set, and I'll need to use tentativeWinningPayoutNumerators  instead, correct?

Reply:
thats correct. Actually since these are finalized or awaiting finalization you could always just use that

Reply:
I'm still unclear on how to get the number of shares in certain cases. For example, if the user is short a particular outcome, they will have a net position that's negative, so that should be considered a positive number of shares in all other outcomes? How should scalar markets be handled?

Reply:
You can get balances from the users TokenBalanceChanged db. The records have a token type and a market so you can get the current balance of all shares for a market.